chrome/content/zotero/tabs.js

```diff
@@ -83,6 +83,10 @@ var Zotero_Tabs = new function () {
 			selected: tab.id == this._selectedID,
 			iconBackgroundImage: tab.iconBackgroundImage
 		})));
+		// Disable File > Close menuitem if multiple tabs are open
+		const multipleTabsOpen = this._tabs.length > 1;
+		document.getElementById('menu_close').setAttribute('disabled', multipleTabsOpen);
+		document.getElementById('menu_close').setAttribute('command', multipleTabsOpen ? false : 'cmd_close');
```

Comment:
You should be able to just disable the command, which will automatically disable associated menu items.

https://docs.huihoo.com/mozilla/xul/xultu/commands.html

Reply:
Right! Still catching up on xulrunner